qemu/tests/seabios.py

```diff
@@ -62,18 +63,24 @@ def info_check():
             err_msg += " is %s" % get_output(vm.serial_console)
             raise error.TestFail(err_msg)
 
+    if not (boot_menu_hint and utils_misc.wait_for(boot_menu, timeout, 1)):
+        raise error.TestFail("Could not get boot menu message.")
+
     if restart_key:
         error.context("Restart vm and check it's ok", logging.info)
 
-        if not (boot_menu_hint and utils_misc.wait_for(boot_menu, timeout, 1)):
-            raise error.TestFail("Could not get boot menu message.")
-
         seabios_text = get_output(seabios_session)
         headline = seabios_text.split("\n")[0] + "\n"
         headline_count = seabios_text.count(headline)
 
         vm.send_key(restart_key)
 
+        if not (boot_menu_hint and utils_misc.wait_for(boot_menu, timeout, 1)):
+            raise error.TestFail("Could not get boot menu message.")
+        # Send boot menu key in monitor.
+        time.sleep(1)
```

Comment:
The `sleep(1) `here can be removed too, i think.

Reply:
removed. It still works.